src/app/components/ModalContainer/pokemonSearchDetail/index.tsx

```
@@ -7,114 +7,114 @@ import { Typography } from "../../atoms/Typography";
+			>
+				<p className="mb-4">
+					<Typography variant="h3" weight="bold" color="black">
+						特性
+					</Typography>
+				</p>
+				<ul className="space-y-4">
+					{pokemonData.abilities.map((ability) => (
+						<li
+							key={ability.name}
+							tabIndex={0}
+							role="article"
```

Biome (1.9.4)

[error] The HTML element li is non-interactive. Do not use tabIndex.

Adding non-interactive elements to the keyboard navigation flow can confuse users.
Unsafe fix: Remove the tabIndex attribute.


(lint/a11y/noNoninteractiveTabindex)